pytorch_pfn_extras/training/extensions/print_report_notebook.py

```diff
@@ -53,6 +53,7 @@ def widget(self) -> HTML:
 
     def __call__(self, manager: _BaseExtensionsManager) -> None:
         log_report = self.get_log_report(manager)
+        log_report._log.register_looker('print_report')
```

Comment:
Please consider the case that the multiple instances of the same extensions are registered.